src/marks/vscode_settings.py

```diff
@@ -0,0 +1,58 @@
+import json
```

Comment:
This file should prob make its way into knausj at some point, but for now leaving it here